src/ynEIGEN/YnEigenFactory.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: BSD-3-Clause License
```

Comment:
if we can please relocate YnEigenFactory and YnEigenStructs 

to script/ynEigen

Since we won't count it as part of core protocol audited code

Reply:
done